ssi_expander.py

```diff
@@ -19,12 +19,12 @@ def process(source, dest):
     dest_dir = os.path.realpath(os.path.join(dest, os.path.relpath(dirpath, source)))
     os.mkdir(dest_dir)
     for filename in filenames:
-      src_path = os.path.abspath(os.path.join(source, dirpath, filename))
+      src_path = os.path.abspath(os.path.join(dirpath, filename))
       dest_path = os.path.join(dest_dir, filename)
       if not filename.endswith('.html'):
         os.symlink(src_path, dest_path)
       else:
-        file(dest_path, 'w').write(ssi.InlineIncludes(src_path))
+        file(dest_path, 'w').write(ssi.InlineIncludes(src_path, "/%s" % os.path.relpath(src_path)))
```

Comment:
It is because [this line](https://github.com/danvk/ssi-server/blob/master/ssi.py#L21):
```
file_to_read = os.path.join(os.path.dirname(web_path),file_to_read)[1:]
```
InlineIncludes function waits leading '/' because ssi_server invoke it with relative url path which includes leading '/'. Whats why `[1:]` here, so without `"/%s" % ` it just truncates the fist symbol of the path.